schemas/json/theme.json

```diff
@@ -1145,7 +1145,30 @@
 			"type": "object",
 			"properties": {
 				"button": {
-					"$ref": "#/definitions/stylesPropertiesComplete"
+					"type": "object",
+					"allOf": [
+						{
+							"$ref": "#/definitions/stylesProperties"
+						},
+						{
+							"properties": {
+								"border": {},
+								"color": {},
+								"spacing": {},
+								"typography": {},
```

Comment:
Are `filter`, `outline`, and `shadow` intentionally excluded from here?

Reply:
Not intentionally! This was based entirely on the `link` settings and those can be added in.

Reply:
Pushed a commit to allow those again 👍